app/views/auctions/single_bid/_auction_status.html.erb

```diff
@@ -1,3 +1,7 @@
-<span class="bid-amount">
-<%= @view_model.auction_user_bid_amount_as_currency %>
-</span>
+<% if @view_model.auction_won? %>
+  <%= render partial: "../multi_bid/auction_status" %>
```

Comment:
This is indeed ugly, but it's to avoid a bunch of functions instead of template_path, but we might reverse it.